plugins/org.locationtech.udig.catalog.teradata/src/org/locationtech/udig/catalog/teradata/Activator.java

```diff
@@ -318,7 +316,8 @@ protected Button createButton(Composite parent,
 
 				public File findPluginsDir() throws IOException {
 					Bundle bundle = Platform.getBundle(GEOTOOLS_LIBS_PLUGIN);
-					String filePath = FileLocator.toFileURL(FileLocator.find(bundle, new Path("gtlib"), emptyMap())).getFile();
+					
+					String filePath = FileLocator.toFileURL(FileLocator.find(bundle, new Path("glib"), new HashMap<String, String>())).getFile();
```

Comment:
why the path has changed from "gtlib" to "glib"?

Reply:
Good question. Just had a look at the complete class and it looks like it doesn't work anyway. The given bundle defined in `GEOTOOLS_LIBS_PLUGIN` doesn't exists anymore, does it?

Reply:
To be honest I have not worked with teradata. The bundle specified by  GEOTOOLS_LIBS_PLUGIN  seems to be absent